src/asciidoc/docs/develop/editor.adoc

```diff
@@ -36,30 +36,44 @@ For in-depth walkthroughs of working outside the Koji editor, see <<use-git#>>.
 
 == Previews
```

Comment:
Let's add a link to the "Testing" page in this section. Eventually, we'll want to rework this page entirely to be more about "developing in the Koji editor", but that's a project for another day!